org.simondalvai.ball2box.yml

```
@@ -0,0 +1,38 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: '3.5'
```

```suggestion
runtime-version: '23.08'
sdk: org.freedesktop.Sdk
base: org.godotengine.godot.BaseApp
base-version: '3.5-23.08'
```

This provide a more modern runtime.